11ty/documentation/development.md

```diff
@@ -5,44 +5,36 @@ parent:
   href: '/documentation'
 ---
 
-This guide will walk you through the current state of the development config. Things get a bit technical here. 🤖
+This guide will walk you through the process of setting up your computer so that you can run Self-Defined locally.
 
-All file names, except stated differently, are relative to the project root.
+## Pre-requisites
 
-We are using [Eleventy](https://www.11ty.io/) to generate the site based on the given definitions. As a module bundle we use Parcel.
+To contribute to Self-Defined, you should be familiar with local development tools such as Git and the terminal, and comfortable working in a text editor such as Visual Studio Code, Sublime, or Atom. Additionally, you'll need to a GitHub account to submit your changes to us for review.
 
-## Setup
+We're working on making it possible to contribute definitions to Self-Defined without the need for developer tooling! Stay tuned!
 
-You will need to have Node and Git installed on your machine. To start developing please create a fork of the root repository.
+## Installing Self-Defined
 
-Afterwards, install the dependencies.
+In your terminal, run the following command:
 
 ```bash
 npm install
 ```
 
-That’s it, actually.
+This will install everything you need to run Self-Defined locally.
 
-## Development Mode
+## Starting the development server
 
-To start the dev server run this command:
+In your terminal, run the following command:
 
 ```bash
 npm run serve
 ```
 
-This will start Eleventy’s development server and watch the CSS entry point for changes. The output in the console will show the local URL to use.
+This will start a local development server. Once the startup process completes, you your terminal will show you the local URL that you can visit to view Self-Defined in your browser. We default to [http://localhost:8080](http://localhost:8080).
 
-### Eleventy Config
+The `serve` script watches your files for changes and then refreshes your browser. Try `assets/css/abstracts/_variables.scss` and changing a color variable defined there. You'll see your new color as soon as you save!
 
-Configuration for Eleventy (e.g. filters and collections) are defined in `.eleventy.js`.
+## About the project
 
-## Build
-
-To build the production site run
-
-```bash
-npm run build
-```
-
-This will generate a minified CSS bundle as well as run Eleventy’s production build. Build files are located in `dist`.
+We're using [Eleventy](https://www.11ty.io/) to generate the files that make up the Self-Defined website, and [Sass](https://sass-lang.com) to write our CSS. You're welcome to make contributions to any part of this code base. If you're not sure where to begin, check out [our issues on GitHub](https://github.com/tatianamac/selfdefined/issues)!
```

Comment:
I think instead of revealing the Front Matter Data at the top tier of the documetnation, that should come as a sub-tier of this documentation page. Following this "About the Project" would be perfect.

I'd like to add something like:

> We're using [Eleventy](https://www.11ty.io/), a static site generator, to generate the files that make up the Self-Defined website, and [Sass](https://sass-lang.com) to write our CSS. You're welcome to make contributions to any part of this code base. If you're not sure where to begin, check out [our issues on GitHub](https://github.com/tatianamac/selfdefined/issues)! If you're having trouble, feel free to open an issue!

Static site generators like Eleventy rely on something called Front Matter. Front Matter makes it such that all of the content can be written in Markdown then built into the static HTML and CSS. Here's how we structured our [front matter](/front-matter/).